docs/source/getting_started/installation.rst

```diff
@@ -85,3 +85,9 @@ You can also build and install vLLM from source:
 
         $ nvcc --version # verify that nvcc is in your PATH
         $ ${CUDA_HOME}/bin/nvcc --version # verify that nvcc is in your CUDA_HOME
+
+    Sometimes, forcing a reinstall and specifying the CUDA Toolkit path can help resolve build errors.
+
+    .. code-block:: console
+
+        $ CUDACXX=/usr/local/cuda-12/bin/nvcc CMAKE_ARGS="-DLLAMA_CUBLAS=on -DCMAKE_CUDA_ARCHITECTURES=native" FORCE_CMAKE=1 pip install -e . --no-cache-dir --force-reinstall --upgrade
```

Comment:
This command is too specfic. I suggest delete it. Just remind user to re-install in a fresh new environment.

Reply:
Good point. Maybe just hint the force reinstall flag so users don't need to look it up.

Reply:
@youkaichao ok, changed it now